source/renderer/app/components/widgets/forms/ReadOnlyInput.js

```diff
@@ -21,10 +22,17 @@ export default class ReadOnlyInput extends Component<Props> {
   };
 
   render() {
-    const { label, value, onClick } = this.props;
+    const { label, value, onClick, isSet } = this.props;
     const { intl } = this.context;
 
-    const mainClasses = classnames([styles.component, 'changeLabel']);
+    const buttonLabel = intl.formatMessage(
```

Comment:
This makes this component to be usable only in this context. Wouldn't it be better to either rename the component to something `PasswordSetInput` or have the label to be received as a prop?

Reply:
@daniloprates I reused logic we had before. But it would be better to have something like you said. Because we don't have enough time right now and because `<ReadOnlyInput>` is used just in this case I decided to leave that as is to speed up the merging process for this PR.